Tensile/Common.py

```diff
@@ -1292,6 +1291,7 @@ def getArchitectureName(gfxName):
     "MinVgprNumber":                list(range(0,256)),
 
     "MaxVgprNumber":                list(range(0,257)),
+    "MinKForGSU":                   [16,32,64,128,256] # min K size to use GlobalSplitU algorithm (only for HPA now)
```

Comment:
Please move this comment to the above line like other parameters.
Also, we do not need "(only for HPA now)" part.
We should be able to use this for non HPA scenarios (SGEMM, DGEMM, etc...).

Reply:
> Please move this comment to the above line like other parameters. Also, we do not need "(only for HPA now)" part. We should be able to use this for non HPA scenarios (SGEMM, DGEMM, etc...).

That comment is copied from original comment. I don't know why it is 'only for HPA now'.  It should work for non HPA scenarios as you commented.

Reply:
> > Please move this comment to the above line like other parameters. Also, we do not need "(only for HPA now)" part. We should be able to use this for non HPA scenarios (SGEMM, DGEMM, etc...).
> 
> That comment is copied from original comment. I don't know why it is 'only for HPA now'. It should work for non HPA scenarios as you commented.

I understand that you copied it from the original code.
However, if it is not true, we should fix it.